.github/workflows/linux_arm64.yml

```diff
@@ -0,0 +1,1473 @@
+#    # container:
+#    #   image: openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04
+#    #   volumes:
+#    #     - /mount/caches:/mount/caches
+#    env:
+#      OPENVINO_REPO: ${{ github.workspace }}/openvino
+#      INSTALL_DIR: ${{ github.workspace }}/install
+#      INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
+#      MODEL_HUB_TESTS_INSTALL_DIR: ${{ github.workspace }}/install/tests/model_hub_tests
+#    if: fromJSON(needs.smart_ci.outputs.affected_components).TF_FE.test ||
+#        fromJSON(needs.smart_ci.outputs.affected_components).TFL_FE.test
+#
+#    steps:
+#      - name: Check sudo
+#        run: if [ "$(id -u)" -eq 0 ]; then apt update && apt --assume-yes install sudo; fi
+#
+#      - name: Download OpenVINO package
+#        uses: actions/download-artifact@v3
+#        with:
+#          name: openvino_package_x86_64
+#          path: ${{ env.INSTALL_DIR }}
+#
+#      - name: Download OpenVINO tests package
+#        uses: actions/download-artifact@v3
+#        with:
+#          name: openvino_tests_x86_64
+#          path: ${{ env.INSTALL_TEST_DIR }}
+#
+#      - name: Extract OpenVINO packages
+#        run: |
+#          pushd ${INSTALL_DIR}
+#            tar -xzf openvino_package.tar.gz -C ${INSTALL_DIR}
+#          popd
+#
+#          pushd ${INSTALL_TEST_DIR}
+#            tar -xzf openvino_tests.tar.gz -C ${INSTALL_DIR}
+#          popd
+#
+#      - name: Fetch setup_python action
+#        uses: actions/checkout@v4
+#        with:
+#          sparse-checkout: |
+#            .github/actions/setup_python/action.yml
+#          sparse-checkout-cone-mode: false
+#          path: 'openvino'
+#
+#      - name: Setup Python ${{ env.PYTHON_VERSION }}
+#        uses: ./openvino/.github/actions/setup_python
+#        with:
+#          version: ${{ env.PYTHON_VERSION }}
+#          should-setup-pip-paths: 'false'
+#          self-hosted-runner: 'false'
+#
+#      - name: Install OpenVINO Python wheels
+#        run: python3 -m pip install ${INSTALL_DIR}/tools/openvino-*
+#
+#      - name: Install TF Hub tests requirements
+#        run: |
+#          python3 -m pip install -r ${MODEL_HUB_TESTS_INSTALL_DIR}/tf_hub_tests/requirements.txt
+#
+#      - name: TensorFlow Hub Tests - TF FE
+#        run: |
+#          export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
+#          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/tf_hub_tests/ -m ${TYPE} --html=${INSTALL_TEST_DIR}/TEST-tf_hub_tf_fe.html --self-contained-html -v
+#        env:
+#          TYPE: ${{ github.event_name == 'schedule' && 'nightly' || 'precommit'}}
+#          TEST_DEVICE: CPU
+#
+#      - name: Upload Test Results
+#        uses: actions/upload-artifact@v3
+#        if: ${{ !cancelled() }}
+#        with:
+#          name: test-results-tensorflow-hub-models
+#          path: |
+#            ${{ env.INSTALL_TEST_DIR }}/TEST*.html
+#          if-no-files-found: 'error'
+
+# TODO: Enable once they are ready for self-hosted runners
```

Comment:
why are they not ready? 
We run such tests on x64

BTW, can we uncomment all pipelines and adding `if: ${{ 'false' }}` to disable them?

Reply:
These jobs are enabled for the x64 yet they run not on the self-hosted runners but on the GitHub-hosted, I believe due to memory/255 issues. I'm unsure whether they could work on the self-hosted arms.
I will do the `if: ${{ 'false' }}`, it looks cleaner, thanks.

Reply:
changed the wording to indicate that the problems are with the machines and not the tests.